DiskUtils.sh

```diff
@@ -162,18 +163,18 @@ format_device() {
 
     case $format_choice in
       1)
-        mkfs.ext4 "$device"
-        log "${GREEN}Device formatted as ext4 filesystem successfully.${NC}"
+        mkfs.ext4 "$disk"
+        log "${GREEN}Disk formatted as ext4 filesystem successfully.${NC}"
         ;;
       2)
-        mkfs.ntfs "$device" || log "${RED}Failed to format device as NTFS. Ensure ntfs-3g is installed.${NC}"
+        mkfs.ntfs "$disk" || log "${RED}Failed to format disk as NTFS. Ensure ntfs-3g is installed.${NC}"
         ;;
       3)
-        mkfs.fat -F32 "$device"
-        log "${GREEN}Device formatted as FAT32 filesystem successfully.${NC}"
+        mkfs.fat -F32 "$disk"
+        log "${GREEN}Disk formatted as FAT32 filesystem successfully.${NC}"
         ;;
       4)
-        mkfs.exfat "$device" || log "${RED}Failed to format device as exFAT. Ensure exfat-utils is installed.${NC}"
+        mkfs.exfat "$disk" || log "${RED}Failed to format disk as exFAT. Ensure exfat-utils is installed.${NC}"
```

Comment:
exfat-utils are deprecated. `exfatprogs` are used now

Reply:
tahnk

Reply:
sorry my comment was a little misleading. you should check using `mkfs.exfat` instead and ensure the output doesn't say like  `Command 'mkfs.exfat' not found, but can be installed with:`